python/server.py

```diff
@@ -24,67 +23,134 @@
 # PLAID_PRODUCTS is a comma-separated list of products to use when initializing
 # Link. Note that this list must contain 'assets' in order for the app to be
 # able to create and retrieve asset reports.
-PLAID_PRODUCTS = os.getenv('PLAID_PRODUCTS', 'transactions')
+PLAID_PRODUCTS = os.getenv('PLAID_PRODUCTS', 'transactions').split(',')
 
 # PLAID_COUNTRY_CODES is a comma-separated list of countries for which users
 # will be able to select institutions from.
-PLAID_COUNTRY_CODES = os.getenv('PLAID_COUNTRY_CODES', 'US')
+PLAID_COUNTRY_CODES = os.getenv('PLAID_COUNTRY_CODES', 'US').split(',')
 
 # Parameters used for the OAuth redirect Link flow.
 #
-# Set PLAID_OAUTH_REDIRECT_URI to 'http://localhost:5000/oauth-response.html'
+# Set PLAID_REDIRECT_URI to 'http://localhost:5000/oauth-response.html'
 # The OAuth redirect flow requires an endpoint on the developer's website
 # that the bank website should redirect to. You will need to configure
 # this redirect URI for your client ID through the Plaid developer dashboard
 # at https://dashboard.plaid.com/team/api.
-PLAID_OAUTH_REDIRECT_URI = os.getenv('PLAID_OAUTH_REDIRECT_URI', '');
-# Set PLAID_OAUTH_NONCE to a unique identifier such as a UUID for each Link
-# session. The nonce will be used to re-open Link upon completion of the OAuth
-# redirect. The nonce must be at least 16 characters long.
-PLAID_OAUTH_NONCE = os.getenv('PLAID_OAUTH_NONCE', '');
+PLAID_REDIRECT_URI = os.getenv('PLAID_REDIRECT_URI', '')
 
-client = plaid.Client(client_id = PLAID_CLIENT_ID, secret=PLAID_SECRET,
-                      public_key=PLAID_PUBLIC_KEY, environment=PLAID_ENV, api_version='2019-05-29')
+client = plaid.Client(client_id=PLAID_CLIENT_ID,
+                      secret=PLAID_SECRET,
+                      environment=PLAID_ENV,
+                      api_version='2019-05-29')
 
 @app.route('/')
 def index():
   return render_template(
-    'index.ejs',
-    plaid_public_key=PLAID_PUBLIC_KEY,
-    plaid_environment=PLAID_ENV,
-    plaid_products=PLAID_PRODUCTS,
-    plaid_country_codes=PLAID_COUNTRY_CODES,
-    plaid_oauth_redirect_uri=PLAID_OAUTH_REDIRECT_URI,
-    plaid_oauth_nonce=PLAID_OAUTH_NONCE,
+    'index.html',
   )
 
 # This is an endpoint defined for the OAuth flow to redirect to.
 @app.route('/oauth-response.html')
 def oauth_response():
   return render_template(
-    'oauth-response.ejs',
-    plaid_public_key=PLAID_PUBLIC_KEY,
-    plaid_environment=PLAID_ENV,
-    plaid_products=PLAID_PRODUCTS,
-    plaid_country_codes=PLAID_COUNTRY_CODES,
-    plaid_oauth_nonce=PLAID_OAUTH_NONCE,
+    'oauth-response.html',
```

Comment:
Just wondering if we've tested out that the oauth flow works here

Reply:
yes